padd.sh

```diff
@@ -279,7 +279,6 @@ GetSystemInformation() {
     if [ -z "$sys_model" ]; then
       sys_model="${product_family} ${product_name}"
     fi    
```

Comment:
There is one more test failing (if you click on "Details" on the failing test you will be able to see it). 

There are trailing spaces here.

Reply:
Oh, thanks. Now it should be fixed.